app/renderer/actions/main.ts

```diff
@@ -0,0 +1,11 @@
+import { actionCreator } from "./helpers"
+import { ComputedTransaction } from "app/renderer/prefilledTransactions"
+import * as Actions from "./actionNames"
+import {
+  SaveTransactionsAction,
+} from "./actionTypes"
+
+// Action creator to save wallet
+export const saveTransactions = (transactions: ComputedTransaction): SaveTransactionsAction => {
+  return actionCreator<ComputedTransaction>(Actions.SAVE_TRANSACTIONS)(transactions)
+}
```

Comment:
This whole file can be removed as it just implements a function that is already defined in [here](https://github.com/witnet/sheikah/pull/429/files#diff-21f3a335cf1f229255860adc851fe790)